at_commons/lib/src/keystore/at_key.dart

```diff
@@ -190,11 +192,13 @@ class Metadata {
   DateTime? updatedAt;
   String? dataSignature;
   String? sharedKeyStatus;
-  bool? isPublic = false;
+  bool isPublic = false;
   bool isHidden = false;
   bool namespaceAware = true;
   bool? isBinary = false;
-  bool? isEncrypted;
+  @Deprecated(
+      'As of every key-values are encrypted by default, this is no longer required.  It\'s value is always true')
+  bool isEncrypted = true;
```

Comment:
please verify whether setting default value to true will have an impact on certain keys in server which are not encrypted
e.g privatekey: private:

Reply:
Got it

Reply:
Yeah the most challenging part of this change is looking at *every* usage of isEncrypted and for each usage, checking that there is a correct way to achieve the same *without* checking isEncrypted

Also the comment isn't entirely accurate; the point as per original issue which discusses this is that, by default, all data is encrypted unless isPublic is true - ie isPublic gives us all the info we need as to whether the data is encrypted or not

So isEncrypted (the Boolean property) isn't "defaulting to true" - it's just unnecessary, hence being deprecated, hence we need to check everywhere it's being used to verify our idea that it can be deprecated

Reply:
(Note: by "every usage" I mean every usage across all of the repos. Deprecation is tricky complicated work!)

Reply:
PS I'm happy to pair with you for an hour or so Minnu, to do that usages review.

Reply:
Sure Gary, We can schedule a call tomorrow[Feb 10] in your free time.